create_jenkins_job.py

```diff
@@ -264,15 +263,16 @@ def create_job(os_name, job_name, template_file, additional_dict):
             })
 
         # configure nightly triggered job
-        job_name = 'nightly_' + job_os_name + '_debug'
-        if os_name == 'windows':
-            job_name = job_name[:15]
-        create_job(os_name, job_name, 'ci_job.xml.em', {
-            'cmake_build_type': 'Debug',
-            'disabled': os_name == 'linux-armhf',
-            'time_trigger_spec': PERIODIC_JOB_SPEC,
-            'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
-        })
+        if os_name != 'linux-armhf':
+            job_name = 'nightly_' + job_os_name + '_debug'
+            if os_name == 'windows':
+                job_name = job_name[:15]
+            create_job(os_name, job_name, 'ci_job.xml.em', {
+                'cmake_build_type': 'Debug',
+                'disabled': False,
```

Comment:
I would guess this line isn't necessary anymore.

Same below.

Reply:
Good point.  Removed in ea4fd7c.